### ISSUE TITLE
page breaks on h2 headings, not just h1 headings

### ISSUE THREAD

**Issue by nicholascloud** [GitbookIO/gitbook#463]
From what I can gather, gitbook instructs ebook-convert to break pages when the HTML class `book-chapter` is encountered. 

``` javascript
// lib/generate/ebook/index.js
{
  // ...
  "--chapter": "descendant-or-self::*[contains(concat(' ', normalize-space(@class), ' '), ' book-chapter ')]",
  "--chapter-mark": "pagebreak",
  // ...
}
```

Gitbook places this class on `<h1>` elements, as it should. But when I generate pdf, mobi, and epub documents, they break on both `<h1>` and `<h2>` elements (not `<h3>` or lower). In the image below, both headings are `<h2>` elements, and I confirmed that they do not possess the `book-chapter` class by examining the temporary HTML output.

![page breaks on h2 tags](https://i.imgur.com/AaZixFO.png)

This seems to be similar to the default behavior of ebook-convert (http://manual.calibre-ebook.com/cli/ebook-convert.html#cmdoption-ebook-convert--chapter).

**Comment by rapodaca** [comment]
Was this ever resolved?

**Comment by rwestergren** [comment]
Looks like it was patched for [4.0.0-alpha.6](https://github.com/GitbookIO/gitbook/commit/6ea5b2b539b107fff094689d0844e661e3260130).
